cmd/stcrashreceiver/stcrashreceiver.go

```diff
@@ -122,7 +160,7 @@ func (r *crashReceiver) servePut(base string, w http.ResponseWriter, req *http.R
 	if r.dsn != "" {
 		go func() {
 			// There's no need for the client to have to wait for this part.
-			if err := sendReport(r.dsn, path, bs); err != nil {
+			if err := sendReport(r.dsn, base, bs); err != nil {
```

Comment:
I can't see it in the PR if this has an effect, but why is this changing?

Reply:
This is because previously it used the path to construct the url like `$baseURL/aa/bb/aabbccdd...` now it just does `$baseURL/aabbccdd` without the leading directories, as that part is anyway handled magically in the HTTP server.